docker/k8s/vtexplain/Dockerfile

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM vitess/base AS base
+ARG VT_BASE_VER=latest
+
+FROM vitess/base:${VT_BASE_VER} AS base
```

Comment:
For some reason, using `lite` as base doesn't work.
If one of you (@dkhenry / @derekperkins) knows how to fix that, let me know.
I'd like to avoid using `vitess/base` if possible.

Reply:
`lite` doesn't contain `vtexplain` in it, AFAIK. Before I discovered the `vitess/vtexplain` image, I had to set up my ` vtctld` component with `vitess/base` and `kubectl exec` into it to run `vtexplain`, because that binary wasn't available in `/vt/bin` of `vitess/lite`.

Reply:
that explains it (no pun intended) :)